program_data/standard_utility_allowances.yaml

```diff
@@ -0,0 +1,33 @@
+# NOTES: This YAML file is currently incomplete; I am adding each state's data
+# as needed rather than trying to add every state at once. Some states have
+# more complexity, such as multiple regions in one state, and/or different
+# SUA values for different household sizes.
+#
+# SOURCES:
+#
+# "Standard Utility Allowances", USDA:
+#
+# https://www.fns.usda.gov/snap/eligibility/deduction/standard-utility-allowances
+#
+# See the standard_utility_allowances folder and associated README for more
+# information and context.
+
+IL:
+    HEATING_AND_COOLING_UTILITY_ALLOWANCE: $466
+    BASIC_LIMITED_UTILITY_ALLOWANCE: $319
+    ELECTRICITY: $72
+    GAS_AND_FUEL: $72
+    PHONE: $30
+    SEWAGE: $72
+    TRASH: $72
+    WATER: $72
```

Comment:
These amounts (from the USDA/FNS spreadsheet) agree with what I see here — [guidance IDHS, 9/6/18](https://www.dhs.state.il.us/page.aspx?item=110481) — but are different than the amounts that the Legal Aid Chicago calculator uses. Since these values match to two different data sources, I think they're a good bet.